docs/understanding-airbyte/configuring-airbyte.md

```diff
@@ -0,0 +1,109 @@
+4. `JOB_MAIN_CONTAINER_CPU_LIMIT` - Define the job container's maximum CPU usage. Units follow either Docker or Kubernetes, depending on the deployment. Defaults to none.
+5. `JOB_MAIN_CONTAINER_MEMORY_REQUEST` - Define the job container's minimum RAM usage. Units follow either Docker or Kubernetes, depending on the deployment. Defaults to none.
+6. `JOB_MAIN_CONTAINER_MEMORY_LIMIT` - Define the job container's maximum RAM usage. Units follow either Docker or Kubernetes, depending on the deployment. Defaults to none.
+
+#### Logging
+1. `LOG_LEVEL` - Define log levels. Defaults to INFO. This value is expected to be one of the various Log4J log levels.
+
+#### Worker
+1. `MAX_SPEC_WORKERS` - Define the maximum number of Spec workers each Airbyte Worker container can support. Defaults to 5.
+2. `MAX_CHECK_WORKERS` - Define the maximum number of Check workers each Airbyte Worker container can support. Defaults to 5.
+3. `MAX_SYNC_WORKERS` - Define the maximum number of Sync workers each Airbyte Worker container can support. Defaults to 5.
+4. `MAX_DISCOVER_WORKERS` - Define the maximum number of Discover workers each Airbyte Worker container can support. Defaults to 5.
+
+#### Scheduler
+1. `SUBMITTER_NUM_THREADS` - Define the maximum number of concurrent jobs the Scheduler schedules. Defaults to 5.
+2. `MINIMUM_WORKSPACE_RETENTION_DAYS` - Defines the minimum configuration file age for sweeping. The Scheduler will do it's best to now sweep files younger than this. Defaults to 1 day.
+3. `MAXIMUM_WORKSPACE_RETENTION_DAYS` - Defines the oldest un-swept configuration file age. Files older than this will definitely be swept. Defaults to 60 days.
+4. `MAXIMUM_WORKSPACE_SIZE_MB` - Defines the workspace size sweeping will continue until. Defaults to 5GB.
+
+#### Container Orchestrator
+1. `CONTAINER_ORCHESTRATOR_ENABLED` - Define if Airbyte should use Scheduler V2.
+
+### Docker-Only
+1. `WORKSPACE_DOCKER_MOUNT` - Defines the name of the Airbyte docker volume.
+2. `DOCKER_NETWORK` - Defines the docker network the new Scheduler launches jobs on.
+
+### Kubernetes-Only
+#### Jobs
+1. `JOB_KUBE_TOLERATIONS` - Define one or more Job pod tolerations. Tolerations are separated by ';'. Each toleration contains k=v pairs mentioning some/all of key, effect, operator and value and separated by `,`.
+2. `JOB_KUBE_NODE_SELECTORS` - Define one or more Job pod node selectors. Each kv-pair is separated by a `,`.
+3. `JOB_KUBE_MAIN_CONTAINER_IMAGE_PULL_POLICY` - Define the Job pod connector image pull policy.
+4. `JOB_KUBE_MAIN_CONTAINER_IMAGE_PULL_SECRET` - Define the Job pod connector image pull secret. Useful when hosting private images.
+5. `JOB_KUBE_SOCAT_IMAGE` - Define the Job pod socat image.
+6. `JOB_KUBE_BUSYBOX_IMAGE` - Define the Job pod busybox image.
+7. `JOB_KUBE_CURL_IMAGE` - Define the Job pod curl image pull.
+8. `JOB_KUBE_NAMESPACE` - Define the Kubernetes namespace Job pods are created in.
+
```

Comment:
I realised I left out the Kubernetes logging variables - will add them tomorrow.